Sources/Yams/Constructor.swift

```diff
@@ -6,6 +6,8 @@
 //  Copyright (c) 2016 Yams. All rights reserved.
 //
 
+// swiftlint:disable file_length
```

Comment:
A future PR could easily break up this file into `Constructor`, `ScalarConstructible` & `SexagesimalConvertible`. Don't do it here, but it's something that could be done later.